.github/workflows/load-tests-job-configs/config_FnApiRunner_Python_Microbenchmark.txt

```diff
@@ -0,0 +1,26 @@
+###############################################################################
+#  Licensed to the Apache Software Foundation (ASF) under one
+#  or more contributor license agreements.  See the NOTICE file
+#  distributed with this work for additional information
+#  regarding copyright ownership.  The ASF licenses this file
+#  to you under the Apache License, Version 2.0 (the
+#  "License"); you may not use this file except in compliance
+#  with the License.  You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  See the License for the specific language governing permissions and
+# limitations under the License.
+###############################################################################
+--publish_to_big_query=true
+--influx_measurement=python_direct_microbenchmarks
+--project=apache-beam-testing
+--metrics_dataset=load_test
+--metrics_table=python_direct_microbenchmarks
+--input_options={}
+--influxDatabase=beam_test_metrics
```

Comment:
same as https://github.com/apache/beam/pull/28450#discussion_r1333171697 a load test builder step to set common options (e.g., currently if influxHost has changed then it needs to change all files)

Reply:
Done